src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -341,27 +341,104 @@ export class PatientViewPageStore
                     sampleIdToTumorType[mutation.sampleId]);
             }), "id");
 
-            return new Promise((resolve, reject) => {
-                const oncokbSearchPromise = oncokbClient.searchPostUsingPOST(
-                    {body: generateEvidenceQuery(queryVariants)});
+            const oncokbSearchPromise = oncokbClient.searchPostUsingPOST(
+                {body: generateEvidenceQuery(queryVariants)});
+
+            const evidenceLookupPromise = oncokbClient.evidencesLookupPostUsingPOST(
+                {body: generateEvidenceQuery(queryVariants)});
 
+            return new Promise((resolve, reject) => {
                 // TODO return type is not correct for the auto-generated API!
                 // generated return type is Array<IndicatorQueryResp>,
                 // but the actual return type is {meta: {} data: Array<IndicatorQueryResp>}
                 // that's why here we need to force data type to be any and get actual data by data.data
-                oncokbSearchPromise.then((data:any) => {
+                Promise.all([oncokbSearchPromise, evidenceLookupPromise]).then((values) => {
                     const oncoKbData:IOncoKbData = {
                         sampleToTumorMap: sampleIdToTumorType,
-                        indicatorMap: generateIdToIndicatorMap(data.data)
+                        indicatorMap: generateIdToIndicatorMap((values[0] as any).data),
+                        evidenceMap: processEvidence((values[1] as any).data)
                     };
 
                     resolve(oncoKbData);
                 });
 
                 oncokbSearchPromise.catch(() => reject());
+                evidenceLookupPromise.catch(() => reject());
+            });
+        }
+    }, {sampleToTumorMap: {}, indicatorMap: {}, evidenceMap: {}});
+
+    readonly pmidData = remoteData<any>({

```

Comment:
I see, this would be an implicit any.  So we should define the correct type instead.